src/cloudai/configurator/cloudai_gym.py

```
@@ -101,12 +101,13 @@ def reset(
         info = {}
         return observation, info
 
-    def step(self, action: Any) -> Tuple[list, float, bool, dict]:
+    def step(self, action: Any, constraint_check_reward: float = -1.0) -> Tuple[list, float, bool, dict]:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `action`

(ANN401)